models/vyos/vyos_lldp/vyos_lldp.yaml

```diff
@@ -0,0 +1,74 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+    {
+        'metadata_version': '1.1',
+        'status': ['preview'],
+        'supported_by': 'network'
+    }
+
+NETWORK_OS: vyos
+RESOURCE: lldp
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: vyos_lldp
+  version_added: 2.9
+  short_description: Manage link layer discovery protocol (LLDP) attributes on VyOS devices..
+  description: This module manages link layer discovery protocol (LLDP) attributes on VyOS devices.
+  author:
+     - Rohit Thakur (@rohitthakur2590)
+  options:
+    config:
+      description: The provided link layer discovery protocol (LLDP) configuration.
+      type: list
+      elements: dict
+      suboptions:
+        enable:
```

Comment:
If `enable` is a default state, in that case, this option can be changed to `disable`. Refer `junos_lldp` module